### PR TITLE
Added a project for Shrinkwrap

### PR DESCRIPTION
**Pull request by fzakaria**
Added a project for Shrinkwrap for the CROSS GSoC initative.

[View Rendered](https://github.com/uccross/uccross.github.io/pull/37/files?short_path=0e7a817#diff-0e7a8170342ebf88051058e261b4e21e556fdb44012ae941ceb46a2d49753a36)

### CODE REVIEW

File: projects.md [hunk 1]
```diff
@@ -844,6 +844,21 @@ Specific tasks:
 - Add support for visualizing metadata, such as annotations and labels.
 - Create UI elements for plotting statistical summaries computed in real-time.
 
+### Investigate the dynamic linking landscape
+
+- **Topics:** `Operating Systems` `Compilers` `Linux` `Package Management` `NixOS`
+- **Skills:** Experience with systems programming and Linux familiarity
+- **Difficulty:** Moderate to Challenging
+- **Mentors:** [Farid Zakaria](mailto:fmzakari@ucsc.edu)
```
Comment (fzakaria) [inline projects.md:852]: @trws if you have bandwidth and want to co-mentor let me know.
Thank you for help mentoring me thus far :)

Comment (trws) [reply projects.md:852]: Sure, I'd be happy to.  Always happy to mentor when I get the opportunity. =)

**Review by carlosmalt — CHANGES_REQUESTED** [review]
@fzakaria, Please add a level 2 header for the overall project with some background text.

**Review by carlosmalt — APPROVED** [review]
Nice -- Thank you!

**Pull request merged**